src/main/kotlin/com/github/gradle/android/i18n/AndroidI18nPluginExtension.kt

```diff
@@ -85,7 +86,9 @@ open class AndroidI18nPluginExtension(
      * The file name will be the same as the configured source file.
      */
     fun exportI18nResources() {
-        val basePath = Paths.get(sourceFile).fileName.toString()
+        val formatter = DateTimeFormatter.ofPattern(BASE_OUTPUT_PATH_TIMESTAMP_PATTERN)
+        val timestamp = formatter.format(LocalDateTime.now())
+        val basePath = "$BASE_OUTPUT_PATH_PREFIX$timestamp$BASE_OUTPUT_PATH_SUFFIX"
         val outputFile = File(project.buildDir, basePath)
```

Comment:
If we run the export task N times in a row, we're gonna end up with N output files.
Could it be handy to delete previous generated files? :thinking: 

Something like:
```kotlin
// Remove previous generated files.
project.buildDir
        .listFiles { _, name -> name.matches("^$BASE_OUTPUT_PATH_PREFIX.+$BASE_OUTPUT_PATH_SUFFIX$".toRegex()) }
        ?.forEach(File::delete)
```